### ISSUE TITLE
Cannot use get_client_from_auth_file and get_client_from_cli_profile for GraphRbac

### ISSUE THREAD

**Issue by lmazuel** [Azure/azure-sdk-for-python#2857]
https://stackoverflow.com/questions/51101205/azure-sdk-python-status-code-not-found-graphrbacmanagementclient?noredirect=1#comment89255991_51101205

Because `get_client_from_auth_file` does not set correctly `resource` for GraphRbac

**Comment by lmazuel** [comment]
@davjr96 Yes, realized get_client_from_cli_profile is impacted as well...

**Comment by lmazuel** [comment]
Fixed in azure-common 1.1.13
https://pypi.org/project/azure-common/1.1.13/

You can now simply do that (with no tenant ID)

```python
rbac_client = get_client_from_auth_file(GraphRbacManagementClient,auth_path=AZURE_AUTH_LOCATION)
```

I took this opportunity to fix the CLI version of this method as well.
